src/controllers/translation.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import { MiddleWare } from '../types';
+import { IGBO_TO_ENGLISH_API, MAIN_KEY } from '../config';
+
+interface TranslationMetadata {
+  igbo: string;
+}
+
+interface Translation {
+  translation: string;
+}
+
+/**
+ * Talks to Igbo-to-English translation model to translate the provided text.
+ * @param req
+ * @param res
+ * @param next
+ * @returns English text translation of the provided Igbo text
+ */
+export const getTranslation: MiddleWare = async (req, res, next) => {
+  try {
+    const igboText: string = req.body['igbo'];
+    if (igboText.length == 0) {
+      console.log('Cannot translate empty string');
+      throw new Error('Cannot translate empty string');
+    }
+
+    if (igboText.length > 120) {
```

Comment:
can we assign this `120` to a constant value and then add a comment why it's set to 120?

Reply:
done!